### ISSUE TITLE
[feature request] Adding Timeframe option in  ECHistorical

### ISSUE THREAD

**Issue by Biorix** [michaeldavie/env_canada#57]
Hi,
Is it possible to add timeframe option to ECHistorical ?

I modify the ec_historical.py and test_ec_historical.py like this : 

```
        init_schema = vol.Schema(
            {
                vol.Required("station_id"): int,
                vol.Required("year"): vol.All(
                    int, vol.Range(1840, datetime.today().year)
                ),
                vol.Required("language", default="english"): vol.In(
                    ["english", "french"]
                ),
                vol.Required("format", default="xml"): vol.In(["xml", "csv"]),
                vol.Required("timeframe", default=2): vol.In([1,2,3])
            }
        )

        kwargs = init_schema(kwargs)

        self.station_id = kwargs["station_id"]
        self.year = kwargs["year"]
        self.language = kwargs["language"]
        self.format = kwargs["format"]
        self.timeframe = kwargs["timeframe"]
        self.submit = "Download+Data"
```

Let me know if you want me to make a pull request

**Comment by michaeldavie** [comment]
Thanks, if you could open a PR that would be great.

**Comment by Biorix** [comment]
Voir PR #58

**Comment by michaeldavie** [comment]
Closed via #58
